src/Microsoft.DocAsCode.Build.SchemaDriven/Processors/IProcessContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.SchemaDriven.Processors
+{
+    using Microsoft.DocAsCode.Plugins;
+
+    public interface IProcessContext
+    {
+        IHostService Host { get; }
+        FileModel Model { get; }
```

Comment:
No need as it can already be fetched from Host

---
In reply to: [131305963](https://github.com/dotnet/docfx/pull/1919#discussion_r131305963) [](ancestors = 131305963)